solr/CHANGES.txt

```diff
@@ -488,7 +488,7 @@ Bug Fixes
 
 * SOLR-8319: Fix NPE in pivot facets, add non-Analyzed query method in FieldType. (Houston Putman, Isabelle Giguere)
 
-* SOLR-15843: Update Log4J to 2.16 (Mike Drob, janhoy)
+* SOLR-15843: Update Log4J to 2.17 (Mike Drob, janhoy, weidongkl)
```

Comment:
8.11.1 is released, so you need to file a new JIRA and place a changes entry in the 8.11.2 and 9.0 sections.